index.bs

```diff
@@ -912,16 +912,19 @@ spec: webidl
     add the feature.
   </p>
   <pre class='highlight'>
-    &lt;script&gt;
-      navigator.permissions.query({name:'geolocation'}).then(function(result) {
-        if (result.state == 'granted') {
-          showLocalNewsWithGeolocation();
-        } else if (result.state == 'prompt') {
-          showButtonToEnableLocalNews();
-        }
-        // Don't do anything if the permission was denied.
-      });
-    &lt;/script&gt;
+navigator.permissions.query({ name: "geolocation" }).then(({ state }) => {
```

Comment:
With bikeshed, you can leave things inside `<pre>` indented, and it'll figure out the indentation level.